doc/roadmap_2023.md

```diff
@@ -0,0 +1,193 @@
+# Roadmap for 2023
+
+2022 was an interesting year for Vello. We made good progress on a number of fronts, solving some really tricky problems, and also did a rewrite into WGSL, but did not ship a usable renderer. Oh, and we renamed the project from piet-gpu as well.
+
+2023 promises to be even more interesting. With some focus we can ship a usable renderer. The core team (Raph Levien and Chad Brokaw, with help from Arman Uguray) will be able to do most of the tasks listed here. Help from the community could speed things up or add even more features. We also rely heavily on the wgpu community to build solid GPU infrastructure for our uses, and also expect a nontrivial amount of integration work as people adopt Vello for their projects.
```

Comment:
Optional: to me your first two paragraphs bury the lede, the headline here is that in 2023 we intend to ship a usable renderer, perhaps immediately followed by a very terse summary of what it'll be usable _for_.